packages/aws-cdk/lib/api/cloudformation-deployments.ts

```diff
@@ -447,3 +550,9 @@ export class CloudFormationDeployments {
 function isAssetManifestArtifact(art: cxapi.CloudArtifact): art is cxapi.AssetManifestArtifact {
   return art instanceof cxapi.AssetManifestArtifact;
 }
+
+interface NestedStackTemplates {
+  readonly generatedNestedTemplate: any,
+  readonly deployedNestedStackTemplate: any,
```

Comment:
Why is one property called `generatedNestedTemplate`, while the other is `deployedNestedStackTemplate`?

Seems like `deployedNestedTemplate` is the better name here (the Stack part is kind of obvious).